14/umbraco-cms/extending/customize-backoffice/development-flow/vite-package-setup.md

```diff
@@ -93,7 +101,7 @@ The `build:lib:entry` parameter can accept an array which will allow you to expo
 
 **Build Package**
```

Comment:
Is this meant as a heading?

```suggestion
### Build Package
```

Reply:
I don't know what makes sense here. It was a pre-existing line. What would you suggest?